src/algo/depth-to-rgb-calibration/optimizer.h

```diff
@@ -336,6 +364,8 @@ namespace depth_to_rgb_calibration {
         void sum_per_section(std::vector< double >& sum_weights_per_section, std::vector< byte > const& section_map, std::vector< double > const& weights, size_t num_of_sections);
         void images_dilation(yuy2_frame_data& yuy);
         void gaussian_filter(yuy2_frame_data& yuy);
+        bool check_edges_dir_spread(const z_frame_data & z_data);
```

Comment:
Why pass Z data in if it's a member of the class?
If it gets the Z data then it doesn't need to be a member.
If it's a member it doesn't need to get the Z data.

Let's pick one.